app/app/urls.py

```diff
@@ -371,6 +371,9 @@
     url(r'^extension/?', retail.views.browser_extension_chrome, name='browser_extension'),
     path('how/<str:work_type>', retail.views.how_it_works, name='how_it_works'),
     re_path(r'^tribes', retail.views.tribes, name='tribes'),
+    path('join/<str:handle>/', dashboard.views.join_tribe, name='join_tribe'),
+    path('savetribe/<str:handle>/', dashboard.views.save_tribe, name='save_tribe'),
+    path('tribe/tribeleader/', dashboard.views.tribe_leader, name='tribe_leader'),
```

Comment:
^ would the URL's be

`tribe/<str:handle>/join` 
`tribe/<str:handle>/save`
`tribes/<str:handle>/tribe-leader`

^ thoughts on renaming the URI as such ?
Also since there is a join shouldn't there be a leave?
Or can we group it into a common URI like 
`tribe/<str:handle>/` and have a body ` { join: True }` / `{ join: False}`

Reply:
yes I like your pattern, actually I had plans to look at it, let me change it real quick.
But about having 2 urls to change true/false I think is not needed. 
And about the body they are very diff actually pointing to diff models.